openfermioncirq/trotter/algorithms/split_operator_test.py

```diff
@@ -215,13 +212,13 @@ def test_split_operator_trotter_step_asymmetric():
 │       │           │           │
 Z^0.0   Z^0.0       #2──────────YXXY
 │       │           │           │
-│       #2──────────YXXY^0.333  Z
+│       #2──────────YXXY^0.333  Z^-1
```

Comment:
Report this as a bug. It shouldn't be adding `-1` here.

Reply:
Opened https://github.com/quantumlib/Cirq/issues/1080